hatchet/tests/hpctoolkit_latest.py

```diff
@@ -0,0 +1,199 @@
+# Copyright 2017-2023 Lawrence Livermore National Security, LLC and other
+# Hatchet Project Developers. See the top-level LICENSE file for details.
+#
+# SPDX-License-Identifier: MIT
+
+from hatchet import GraphFrame
+from hatchet.node import Node
+
+
+def test_import_entire_db(data_dir: str) -> None:
+    graphframe = GraphFrame.from_hpctoolkit_latest(f"{data_dir}/hpctoolkit-gamess")
+
+    assert len(graphframe.graph.roots) == 1
+    assert graphframe.graph.roots[0]._hatchet_nid == 1195
+    assert graphframe.graph.roots[0]._depth == 0
+    assert graphframe.graph.roots[0].frame["name"] == 1
+    assert graphframe.graph.roots[0].frame["type"] == "entry"
+
+    assert len(graphframe.dataframe) == 10824
+    assert "name" in graphframe.dataframe.columns
+    assert "realtime (i)" in graphframe.dataframe.columns
+    assert "realtime (e)" in graphframe.dataframe.columns
```

Comment:
@draganagrbicllnl I'd hold off on the `time` mapping for now until we hear back from @pearce8. As for the `(i)` and `(e)` part, it would probably be best to change them to `(inc)` and no suffix respectively. That convention isn't actually from Caliper. It's a Hatchet convention.

Reply:
@ilumsden done

Reply:
@ilumsden @draganagrbicllnl Yes, lets go with (inc) and no suffix to match the Caliper convention.